paasta_tools/api/views/autoscaler.py

```diff
@@ -51,6 +51,11 @@ def update_autoscaler_count(request):
     service = request.swagger_data.get('service')
     instance = request.swagger_data.get('instance')
     desired_instances = request.swagger_data.get('json_body')['desired_instances']
+    if not isinstance(desired_instances, int):
+        error_message = 'The provided body does not have an integer value for "desired_instances": {}'.format(
```

Comment:
Shouldn't this be covered by the schema on parameters?

Reply:
Hmm. It currently isn't and somehow is allowed to be None when I was testing this.
It allowed me to POST nothing and it became None in zk.

Let me see if I can enforce this at the schema level for real.